src/utils/types.ts

```diff
@@ -377,6 +377,7 @@ interface ResponseFull {
   isValid?: boolean | null
   hash?: string // Used in Password plugin
   files?: any[] // Used in FileUpload plugin
+  other?: string | undefined // Used in RadioChoice plugin
```

Comment:
```suggestion
  other?: string // Used in RadioChoice plugin
```

When using `?` you already count for `undefined`

Reply:
Okay, that seems to work, thanks. I could have sworn I was getting a typescript error when I did a `if (other===undefined)` at one point, as though it made a distinction between having the value `undefined` and just being absent. 

I guess not. 🤷‍♂️

082f8c4d